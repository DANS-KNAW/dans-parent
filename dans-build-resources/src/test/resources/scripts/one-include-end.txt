One include at the end of the file.


#include <single-line.txt>