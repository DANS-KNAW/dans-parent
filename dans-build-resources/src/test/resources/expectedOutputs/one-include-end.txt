One include at the end of the file.


line 1